cypress/e2e/authentication/sign-in.cy.js

```diff
@@ -0,0 +1,45 @@
+describe("sign in flow", () => {
+  beforeEach(() => {
+    cy.visit(
+      "https://chingu-dashboard-git-dev-chingu-dashboard.vercel.app/sign-in",
+    );
+  });
+
+  it("should have email and password inputs with no inputs and a disabled Sign In button", () => {
+    // test to verify that the email and password inputs are visible
+    cy.get('input[name="email"]').should("be.visible");
+    cy.get('input[name="password"]').should("be.visible");
+    cy.contains("button", "Sign In").should("be.disabled");
+  });
+
+  it("successfully logs a user in ", () => {
+    cy.get('input[name="email"]').type("jessica.williamson@gmail.com");
+    cy.get('input[name="password"]').type("password");
+
+    // ensure Sign In button is visible and abled before clicking
+    cy.contains("button", "Sign In").should("not.be.disabled");
+    cy.contains("button", "Sign In").should("be.visible");
+    cy.contains("button", "Sign In").click();
+
+    // verify we're back on a dashboard page
+    cy.url().should(
+      "match",
+      /https:\/\/chingu-dashboard-git-dev-chingu-dashboard\.vercel\.app\/dashboard(\/\d+)?/,
+    );
+
+    // TODO: uncomment this once PR is pushed where AuthHeader has a data-testid="nav-dropdown-menu" and retest
```

Comment:
I'm testing to see if the user has sign in successfully and so I want the test to find the nav dropdown element. There wasn't a good attribute to grab to indicate the element is a nav dropdown so I added data-testid. It's specifically for testing so it won't affect the UI.

Reply:
Ok I think I understand the issue now. My only concern with this is querying by test id is discouraged in RTL. If we populate the codebase with this, it may be tempting to use that. What about using data-cy instead? I believe that is unique to cypress and not global like data-testid is.